fhir-persistence/src/test/java/com/ibm/fhir/persistence/search/test/AbstractSearchTokenTest.java

```diff
@@ -463,7 +487,13 @@ public void testSearchToken_Coding_NoSystem_not_in() throws Exception {
     @Test
     public void testSearchToken_Identifier() throws Exception {
         assertSearchReturnsSavedResource("Identifier", "code");
+        assertSearchReturnsSavedResource("Identifier", "CODE");
+        // system is not in registry - treated as not case-sensitive
         assertSearchReturnsSavedResource("Identifier", "http://example.org/identifiersystem|code");
+        assertSearchReturnsSavedResource("Identifier", "http://example.org/identifiersystem|CODE");
+        // system is case-sensitive
+        assertSearchReturnsSavedResource("Identifier-validValueAndSystem", "http://hl7.org/fhir/identifier-use|official");
+        assertSearchDoesntReturnSavedResource("Identifier-validValueAndSystem", "http://hl7.org/fhir/identifier-use|OFFICIAL");
 //        assertSearchReturnsSavedResource("Identifier", "http://example.org/identifiersystem|");
```

Comment:
I guess this one was commented out before, but it might be worth adding a comment to explain why.  I thought we had a placeholder issue open for the fact we don't support `system|` searches, but I can't seem to find it now.

Reply:
Added comment referencing placeholder issue #1409